client/src/components/pages/admin/Opened/index.js

```diff
@@ -1,5 +1,67 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import MaterialTable from 'material-table';
+import { Helmet } from 'react-helmet';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+import backGround from '../../../../assets/images/backgroundDash.svg';
+import DashBar from '../../../dashboard/Tabs';
 
-const Opened = () => <h1>Opened </h1>;
+import './index.css';
+
+const getOpened = async () => {
+  const { data } = (await axios.get('/api/v1/dashboard/applicants')).data;
+  return data.filter((element) => !element.applicationSubmittedDate && element);
+};
+
+const Header = {
+  columns: [
+    { title: 'Name', field: 'fullName', width: '20%' },
+    { title: 'Email', field: 'email', width: '20%' },
+    { title: 'Mobile', field: 'mobileNumber' },
+    { title: 'Age', field: 'age' },
+    { title: 'Academy Location', field: 'location' },
+    { title: 'Address', field: 'address' },
```

Comment:
All of these will be changed to columns like in the design
I mean github account, code wars, free code camp and project 
All of these will be only a word and when the admin click on them it will redirect him to the link for each .
https://user-images.githubusercontent.com/29041512/77910871-862e5400-7298-11ea-9289-b97018b29e57.png